src/RepoM.ActionMenu.Core/ActionMenu/Model/ActionMenus/BrowseRepository/RepositoryActionBrowseRepositoryV1Mapper.cs

```diff
@@ -21,25 +20,28 @@ protected override async IAsyncEnumerable<UserInterfaceRepositoryActionBase> Map
             yield break;
         }
 
-        var name = await context.RenderStringAsync(action.Name).ConfigureAwait(false);
-        if (string.IsNullOrWhiteSpace(name))
-        {
-            name = "Browse remote";
-        }
+        string menuTitle;
+        //var menuTitle = await context.RenderStringAsync(action.Name).ConfigureAwait(false);
```

Comment:
not sure why this is commented

Reply:
It's no longer gettng that text that way. It's getting it in a different way, depending on whether there is one or more remotes. See lines 32-52.